comeet/UI/MainMenu/MainMenuViewController.swift

```diff
@@ -105,47 +90,27 @@ extension MainMenuViewController : WWCalendarTimeSelectorProtocol {
 }
 
 private extension MainMenuViewController {
+    
     func setup() {
-        
         title = viewModel?.title()
         navigationItem.setHidesBackButton(true, animated: false)
         
-        addGestureRecognizers()
         dateButton.setTitle(viewModel?.selectedDate.displayStringDate(), for: .normal)
         
-        addRoomsList()
-        addAgendaView()
+        self.agendaVC = getAgendaVC()
+        self.roomListVC = getRoomsListVC()
+        if let agendaVC = self.agendaVC {
+            if let roomListVC = getRoomsListVC() {
```

Comment:
You can link this two unwrappings `if let agendaVC = self.agendaVC, let roomListVC = getRoomsListVC()`